started

luisabellan opened the pull request.

luisabellan committed:
started

luisabellan committed:
mvp

luisabellan committed:
mvp

luisabellan committed:
stretch

luisabellan committed:
stretch finished

luisabellan committed:
moved the input away from the button a bit

luisabellan committed:
first part is finished

luisabellan committed:
first part is finished

luisabellan committed:
first part is finished

luisabellan committed:
first part is finished

luisabellan committed:
adding styled components

luisabellan committed:
adding styled components

luisabellan committed:
styling bug fixed

BryceEvans reviewed (approved):
I cloned this down locally and took a look. It looks pretty good! I didn't see any props issues with the way you reorganized it. Nice job!

luisabellan committed:
save

luisabellan committed:
made responsive and added some styling to the styled components too. MVP

luisabellan committed:
final

luisabellan committed:
final commit before stretch

SquareRobin committed:
jest implemented and changed from styled-components to emotion

There were 2 options, @emotion/core and @emotion/syled. The second option required less code as the syntax for the components is the same as the one used in styled-components, only the imports differ

SquareRobin committed:
corrected import

SquareRobin committed:
deployed

https://nasa-photo-of-the-day.squarerobin.now.sh

SquareRobin committed:
Created a test and researched the outcome, findings documented, All streches except for TailWind done

I'm studying TailWind right now.

SquareRobin committed:
Added a catch()  to log out possible errors. cleaned some code

I got a warning about React on line 1 not being used, so I deleted it but kept {useState , useEffect}, Also got rid of semicolons when possible

luisabellan committed:
save

luisabellan committed:
save

luisabellan committed:
save

luisabellan committed:
typos corrected

SquareRobin committed:
changed title

SquareRobin committed:
improved styling

SquareRobin committed:
save

SquareRobin committed:
save

SquareRobin committed:
datapicker re-styled

SquareRobin committed:
save

SquareRobin committed:
save

luisabellan committed:
added some css for very narrow mobile screens

luisabellan committed:
commented out console.log(data)

SquareRobin committed:
changed API key

SquareRobin committed:
API key updated

SquareRobin committed:
updated API_KEY

SquareRobin committed:
changed API key

SquareRobin committed:
Merge branch 'luis-abellan' of https://github.com/luisabellan/nasa-photo-of-the-day into luis-abellan
